test/e2e/service_serving_ca_bundle_exists_test.go

```diff
@@ -0,0 +1,71 @@
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/remotecommand"
+
+	"github.com/kube-reporting/metering-operator/test/reportingframework"
+)
+
+func testReportOperatorServiceCABundleExists(t *testing.T, rf *reportingframework.ReportingFramework) {
+	// Attempt to grab the reporting-operator Pod in the rf.Namespace.
+	// Because this Pod is spun up using a Deployment instead of a Statefulset
+	// we need to list any Pods matching the app=reporting-operator label as we
+	// don't know the name ahead of time.
+	podList, err := rf.KubeClient.CoreV1().Pods(rf.Namespace).List(context.Background(), metav1.ListOptions{
```

Comment:
Is waiting for all pods to have spun up implicit here somehow? Like if we're at this point in the code than that's a given?

Reply:
We wait for all Pods to become "ready" before running any of the post-install tests, so yes that's the assumption.

Reply:
I was trying to find where the pod waiting happens

Reply:
https://github.com/kube-reporting/metering-operator/blob/master/test/deployframework/context.go#L221-L224